src/convertToObject.js

```diff
@@ -7,6 +7,30 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+
+  const declarations = sourceString.split(';');
```

Comment:
Consider handling cases where the input string might end with a semicolon, which would result in an empty string in the `declarations` array. The current implementation correctly skips empty declarations, so this is not a critical issue.